ADJ-Internship/WebApp/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=.;Database=AdjInternship;Trusted_Connection=True;MultipleActiveResultSets=true;"
+    "DefaultConnection": "Server=DESKTOP0105;Database=AdjInternship;Trusted_Connection=True;MultipleActiveResultSets=true;"
```

Comment:
this will not work on other machines, because there is no "DESKTOP0105" on other machines